guides/v2.3/extension-dev-guide/message-queues/config-mq.md

```diff
@@ -49,7 +49,7 @@ The following sample defines two synchronous topics. The first topic is for RPC
 </config>
 ```
 
-### topic element###
+### topic element ###
```

Comment:
You might as well remove the trailing #s from the header. The linting tool will prefer the simpler version, without the trailing #s.